src/services/torrent.rs

```diff
@@ -98,11 +98,16 @@ impl Index {
     /// * Unable to get the category from the database.
     /// * Unable to insert the torrent into the database.
     /// * Unable to add the torrent to the whitelist.
+    /// * Torrent title is too short.
     pub async fn add_torrent(&self, mut torrent_request: AddTorrentRequest, user_id: UserId) -> Result<TorrentId, ServiceError> {
         let _user = self.user_repository.get_compact(&user_id).await?;
 
         torrent_request.torrent.set_announce_urls(&self.configuration).await;
 
+        if torrent_request.metadata.title.len() < 3 {
```

Comment:
Hi @alexohneander maybe we could add a constant for the magic number 3?

```rust
const MIN_TORRENT_TITLE_LENGTH: u32 = 3;
```

Reply:
@josecelano Absolutely. It would actually be nicest if the admin could adjust this dynamically.

Reply:
Hi @alexohneander, yes, that would be the perfect solution, but for now, there is only one way to inject configuration into the application using the config file: `config.toml`.

I discussed with @da2ce7, and [we agreed on the need for two types of configuration](https://github.com/torrust/torrust-index-backend/issues/144#issuecomment-1602340005):

1. `config.toml` file.
2. Configuration in the database

We should put in the `config.toml` only the data required to start the services and/or configuration that does not change very often. This configuration is updated only by sysadmins, that's, people with access to the infrastructure.

We wanted to create a `settings` table with a configuration you can change dynamically and/or you can be defined when you are installing the application. This configuration can only be changed by the website admin.

I'm not sure where this value should go. Once you defined a min length, it would be easy to change it to a greater one, but you cannot do the opposite (unless you truncate the titles), so it seems like a value you have to define during the installation, and you cannot change it. If you @da2ce7 and @alexohneander agree, we can add a new section in the `config.toml` file:

```toml
[index]
min_torrent_title_length = 3
```

or 

```torrent
[index]
min_title_length= 3
```

What do you think?

Anyway, we can merge this PR and open a new issue for that enhancement.

Reply:
Maybe this should be a database configuration that can only be set during the installation. As this restriction is a restriction for dynamically generated database data in the database maybe it makes more sense to add the value in the database. In the future, you could define a way to truncate titles.

Reply:
In summary, we could have three types of configuration values:

1. Infrastructure: `config.toml` or env var.
2. Database read-only. Set during installation or with migrations. It can be defined by the admin during installation.
3. Database read-write. They can be changed by app admins.

But we do not have yet an installation script. The simplest one should generate the initial values for the settings table. For the time being, we could generate the `settings` table and seed it with the default values. The first setup process could be just a console command. In the future, it could be web assistant or console interactive assistant.